amundsen_application/static/js/components/common/ResourceListItem/UserListItem/index.tsx

```diff
@@ -55,12 +55,6 @@ class UserListItem extends React.Component<UserListItemProps, {}> {
             </div>
           </div>
           <div className="resource-type">User</div>
-          <div className="resource-badges">
```

Comment:
We still need this `<div>` and the `<img>`, just remove the `Flag`.

Reply:
My understanding was that in the future Users would not have badges anymore, are there any other badges that are not Alumni for users?

Reply:
Also I don't see the `icon-right` img in the user page at all in prod.

Reply:
1. The decision is not that Users will never have badges, but that the `Alumni` UI would change. Users could still have badges if there is a product requirement for it (not at Lyft but maybe elsewhere). Additionally, there's a case for `Alumni` returning to official badge status in the future, so that we can lower the search ranking for old employees. (Example: I'm not the first person with my name that shows up in user search, but I'm the only active employee 🙃 )
2. The icon shows up on hover.